#################################################################################
#                          Config Files for Python                              #
#################################################################################

vehicle:
  size:
    width: 0.21
    height: 0.22
  velocity: 0.0
camera:
  width: 640
  height: 480

dispatch:
  scene_map: src/agent/map_2

data:
  cam_dir: /dataset/camera_data/
  rad_dir: /dataset/radar_data/
output:
  RadCamFusion_dir: output/RadCamFusion/
  LidCamFusion_dir: output/LidCamFusion/
topic:
  radar_2: /ARS_408_21_2_Topic
  radar_3: /ARS_408_21_3_Topic
visual_assistant:
  base_image: src/utils/visual/scene_base.png
use_cuda: False

calib:
  camera_info_dir: src/camera_info/
  calib_dir: src/utils/calib/
  pole_pose:
    pole1: [-0.415985705489921, 1.11290021179622, 0.481, 1.5707963267949, 0, 0.785398163397456]
    pole2: [-1.59824637808195, -0.790114867663065, 0.461, 1.5707963267949, 0, 0]
    pole3: [0.964315628564147, 0.200335452740542, 0.461, 1.5707963267949, 0, 0]
    pole4: [0.000974248361056473, 0.150707650139425, 0.480970397293368, 1.5707963267949, 0, 0]
  camera_pose:
    11: [-0.33159636363535226, -0.0883799948475499, 0.7180063415966533, -1.5707963267949, 0.785398163397456, 0]
    12: [0.035417733646562054, -0.0800003106380456, 0.8332431804789487, -1.5707963267949, 2.35619449019234, 0]
    13: [0.261482246581381, -0.040821726737309, 0.516250477506555, -1.5707963267949, -2.356194490192345, 0]
    14: [-0.09284049268553585, -0.0420123457579493, 0.2698285834803398, -1.570796326794895, -0.785398163397456, 0]
    2:  [0.298406060019312, -0.0608075188884471, 0.460567976723035, 3.14159265358979, 1.5707963267949, -1.5707963267949]
    3:  [-0.00819043746300863, -0.0558609225821797, 0.201782124653999, -1.570796326794895, 1.570796326794895, 0]
    41: [-0.00110365078598126, -0.145715673177097, 1.25963835404248, 0, -1.570796326794895, -1.570796326794895]
    42: [0.0929564555419943, -0.145715673177097, 1.15083705256435, -1.5707963267949, 0, 0]
    43: [-0.000844845936131017, -0.145715673177097, 1.05677694623637, -1.570796326794895, 1.570796326794895, 0]
    44: [-0.0949049522641072, -0.145715673177097, 1.1505782477145, -1.570796326794895, 3.14159265358979, 0]

lidar_detection:
    cfg_file: /src/LidCamFusion/OpenPCDet/tools/cfgs/custom_models/pointrcnn.yaml
    ckpt_file: /src/LidCamFusion/OpenPCDet/output/custom_models/pointrcnn/01/ckpt/checkpoint_epoch_192.pth

lid_cam_fusion:
  iou_thresh: 0.4
  camera_weight: 0.4
  lidar_weight: 0.6

eval:
  lid_cam_fusion:
    vnums: 1